addons/web/static/src/xml/base.xml

```diff
@@ -1089,33 +1089,46 @@
         </t>
     </div>
 </div>
-<div t-name="FieldBinaryFileUploader.files" class="oe_attachments" aria-atomic="true">
```

Comment:
there is a scss rule in portal based on classname oe_attachments, it is probably broken with this ref

Reply:
I have checked and there is nothing to break with this scss rule in the portal, It's used with different behavior for project module. if I am wrong please correct me

Reply:
Ok you're probably right. But then there is something wrong with this rule. However, it's outside the scope of this task so you can ignore my previous comment